esrally/driver/runner.py

```diff
@@ -951,28 +959,22 @@ async def request_body_query(self, es, params):
 
         r = await self._raw_search(es, doc_type, index, body, request_params, headers=headers)
 
-        if detailed_results:
-            props = parse(r, ["hits.total", "hits.total.value", "hits.total.relation", "timed_out", "took"])
-            hits_total = props.get("hits.total.value", props.get("hits.total", 0))
-            hits_relation = props.get("hits.total.relation", "eq")
-            timed_out = props.get("timed_out", False)
-            took = props.get("took", 0)
-
-            return {
+        stats = {
                 "weight": 1,
                 "unit": "ops",
                 "success": True,
-                "hits": hits_total,
-                "hits_relation": hits_relation,
-                "timed_out": timed_out,
-                "took": took
-            }
-        else:
-            return {
-                "weight": 1,
-                "unit": "ops",
-                "success": True
-            }
+                "request_id": request_id
```

Comment:
Is this a leftover? Because in the next line, the `request_id` is conditionally added to this dict.

Reply:
Well spotted and also a lint issue